src/sqlancer/questdb/gen/QuestDBAlterIndexGenerator.java

```diff
@@ -33,7 +34,7 @@ public static SQLQueryAdapter getQuery(QuestDBGlobalState globalState) {
 
         // We should always choose column with SYMBOL type
         QuestDBColumn columnWithSymbolType = table
-                .getRandomColumnOrBailout(c -> c.getType().toString().equals("SYMBOL"));
+                .getRandomColumnOrBailout(c -> c.getType().getPrimitiveDataType().equals(QuestDBDataType.SYMBOL));
```

Comment:
A small nitpick: enums are typically compared using an equality operator (e.g., see https://stackoverflow.com/a/2937561 for the advantages).